src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -15,18 +15,23 @@
     left: 50%;
   }
 
-  padding: 15px;
-  border-radius: 0 0 50px 50px;
+  padding: $announcement-container-padding;
+  border-radius: 0 0 20px 20px;
   text-align: center;
 
   .close-button {
     cursor: pointer;
     position: absolute;
-    top: 10px;
-    right: 15px;
+    top: 15px;
+    right: 7.5px;
     :hover {
       color: var(--primary-color);
       text-decoration: underline;
     }
+    margin-right: 7.5px;
```

Comment:
Does this have any effect, since it’s positioned absolutely?

Reply:
Yes
Without: 
<img width="539" alt="Screen Shot 2021-02-19 at 6 26 46 PM" src="https://user-images.githubusercontent.com/51083763/108545731-14ac2980-72e0-11eb-9b4f-3a9223a7840e.png">
(the x is a little bit too far to the right)

Reply:
But instead of having a margin, it looks the same if I make ```right```  15px

Reply:
Let’s do the right 15px, as that’s more obvious/clear